### PR TITLE
fixing window undefined error + removing useIsMobile + singleton pris…

### PR DESCRIPTION
**Pull request by luisflores93**
fixing window undefined error + removing useIsMobile + singleton prisma instance

### CODE REVIEW

File: src/helpers/prisma.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// single prisma instance
+import { PrismaClient } from '@prisma/client';
+
+const prismaClientSingleton = () => {
+  return new PrismaClient();
+};
+
+declare global {
```
Comment (fiizzy) [inline src/helpers/prisma.ts]: Curious as to why you're delcaring this?

Comment (luisflores93) [reply src/helpers/prisma.ts]: its just for TypeScript to understand the type of the global variable prisma and allows the TypeScript compiler to provide type checking and suggestions, but i think its not necessary... 🤔 I'll remove it

File: src/app/[user-name]/layout.tsx [hunk 2]
```diff
@@ -9,17 +8,17 @@ export const metadata: Metadata = {
...
-}: {
+} // params,
+: {
   children: React.ReactNode;
   params: { 'user-name'?: string };
 }) {
-  const userId = params?.['user-name'];
-  const user = await getUserDetails(userId);
+  // const userId = params?.['user-name'];
+  // const user = await getUserDetails(userId);
```
Comment (fiizzy) [inline src/app/[user-name]/layout.tsx:17]: Is this comment important?
If so, you can add a `TODO: remove after development`

Comment (luisflores93) [reply src/app/[user-name]/layout.tsx:17]: ahh no actually that will be added when backend were ready

Comment (fiizzy) [reply src/app/[user-name]/layout.tsx:17]: Okay. So do you want to keep it or take it out?

**Review by fiizzy — APPROVED** [review]
lgtm

**Pull request merged**